tests/visualization_tests/matplotlib_tests/test_slice.py

```diff
@@ -28,7 +27,34 @@ def test_plot_slice() -> None:
     assert len(figure.findobj(PathCollection)) == 0
     plt.savefig(BytesIO())
 
-    study = prepare_study_with_trials(with_c_d=False)
+    study = create_study(direction="minimize")
+    study.add_trial(
+        create_trial(
+            value=0.0,
+            params={"param_a": 1.0, "param_b": 2.0},
+            distributions={
+                "param_a": FloatDistribution(0.0, 3.0),
+                "param_b": FloatDistribution(0.0, 3.0),
+            },
+        )
+    )
+    study.add_trial(
+        create_trial(
+            value=2.0,
+            params={"param_b": 0.0},
+            distributions={"param_b": FloatDistribution(0.0, 3.0)},
+        )
+    )
+    study.add_trial(
+        create_trial(
+            value=1.0,
+            params={"param_a": 2.5, "param_b": 1.0},
+            distributions={
+                "param_a": FloatDistribution(0.0, 3.0),
+                "param_b": FloatDistribution(0.0, 3.0),
+            },
+        )
+    )
```

Comment:
We may remove these changes after https://github.com/optuna/optuna/pull/3784 is merged.